Automattic/pocket-casts-android
# Add Total Time story

## Description

This adds total time story UI.

I had to drop "and" between different length sections due to localization nightmare. So "2 days and 10 hours" is "2 days 10 hours" instead and so on.

Figma: lH66LwxxgG8btQ8NrM0ldx-fi-3054_3894

## Testing Instructions

1. Sign in with an account that listened to at least 5 minutes of episodes this year.
2. Sync your profile data.
3. Close the app.
4. Reopen the app.
5. Open Playback 2024.
6. You might see a loading progress bar and it might take some time to sync your data.
7. Once data is synced go to the total time story and verify it.

## Screenshots or Screencast 

![Screenshot 2024-10-17 at 10 48 35](https://github.com/user-attachments/assets/77588a62-07b7-4d3e-842d-600831daa89a)

## Checklist
 
#### I have tested any UI changes...
